src/cpp/desktop/DesktopBrowserWindow.cpp

```diff
@@ -68,10 +68,6 @@ BrowserWindow::BrowserWindow(bool showToolbar,
    setUnifiedTitleAndToolBarOnMac(true);
 
    desktop::enableFullscreenMode(this, false);
-
-   QShortcut* copyShortcut = new QShortcut(QKeySequence::Copy, this);
```

Comment:
Relevant commit for this here: https://github.com/rstudio/rstudio/commit/0a22b50e31f8940bf8a398f4c07701d0e6e0968a

Tested in Ubuntu Desktop and all seems okay, but probably need to think about this a bit more.